jenkins-jobs/git/git-notify-ib-updates

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+
+import datetime
+import json
+import logging
+import sys
+import os
+import urllib.error
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "shift"))
+try:
+    from github_utils import github_api
+    import libib
+
+    # noinspection PyUnresolvedReferences
+    from libib import PackageInfo, ErrorInfo
+except ImportError:
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "shift"))
+    from github_utils import github_api
+    import libib  # NOQA
+
+    # noinspection PyUnresolvedReferences
+    from libib import PackageInfo, ErrorInfo
+
+try:
+    current_shifter = libib.fetch("/SDT/shifter.txt", content_type=libib.ContentType.TEXT)
+    exit_code = 0
+except (urllib.error.URLError, urllib.error.HTTPError) as e:
+    print("WARNING: failed to get current_shifter!")
+    print(e)
+    current_shifter = "all"
+    exit_code = 1
+
+cache_root = "/build/builds/cms-ib-notifier"
+
+header = f"@{current_shifter} New IB failures found, please check:"
+
+table_header = """# {rel}-{ib_date} for {arch}
+| Error | Additional data |
+| --- | --- |
+"""
+
+max_report_lines = 19
+
+
+def get_commit_info(repo, commit):
+    return github_api(
+        uri="/repos/{}/commits/{}".format(repo, commit), method="GET", all_pages=False
+    )
+
+
+def isoparse(strDate):
+    return datetime.datetime.strptime(strDate, "%Y-%m-%dT%H:%M:%SZ")
+
+
+def main():
+    workspace = os.getenv("WORKSPACE", os.path.expanduser("~"))
```

Comment:
I would suggest to use pwd as default instead of HOME. HOME on jenkins controller is `/var/lib/jenkins` and we do not want to write any thing in that path

Reply:
Ok